client/src/containers/Topic/Topic/TopicData/TopicData.jsx

```diff
@@ -32,6 +32,7 @@ import { capitalizeTxt, getClusterUIOptions } from '../../../../utils/functions'
 import { setProduceToTopicValues, setUIOptions } from '../../../../utils/localstorage';
 import Select from '../../../../components/Form/Select';
 import * as LosslessJson from 'lossless-json';
+import { Parser } from "json2csv";
```

Comment:
You add a package here but don't install it (since no modification on package lock)

Reply:
Ah I forgot to comment the package lock file

Reply:
you should add the package and package-lock, or it will be break to build